backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/CassandraTableManager.java

```diff
@@ -52,9 +52,9 @@ public InitializationStatus initializeTables(CassandraTypesProvider typesProvide
         KeyspaceMetadata keyspaceMetadata = session.getMetadata().getKeyspaces().get(session.getKeyspace().get());
 
         return module.moduleTables()
-                .stream()
+                .parallelStream()
```

Comment:
it will use the fork/join executor to execute IO base tasks, you should switch to reactor + elastic executor

Reply:
Got lazy, sorry...